orgparse/tests/test_data.py

```diff
@@ -90,3 +90,42 @@ def test_iter_node():
 
     by_iter = [n.heading for n in node]
     assert by_iter == ['H1', 'H2', 'H3']
+
+
+def test_commented_headings_do_not_appear_as_children():
+    root = loads("""
+* H1
+#** H2
+** H3
+#* H4
+#** H5
+* H6
+""")
+    top_level = root.children
+    assert len(top_level) == 2
+
+    h1 = top_level[0]
+    assert h1.heading == "H1"
+    assert h1.get_body() == "#** H2"
```

Comment:
I guess it's also debatable whether get_body should include comments or not, but don't have a strong opinion on it (and anyway not for this pull request)